docs/docs/quick-start.md

```diff
@@ -6,70 +6,101 @@ sidebar_position: 0
 
 Build a multiplayer game for the [Rune platform](https://www.rune.ai) and its millions of players. Rune handles netcode, servers, voice chat, matchmaking, spectating, and much more.
 
-## Install {#install}
-
-Create a new Rune game project by running: 
+Run the command below to get a running multiplayer Tic Tac Toe game!
 
 ```sh
 npx rune-games-cli@latest create
 ```
 
-or follow this [guide to port your existing game](./how-it-works/existing-game.md) to Rune!
+Alternatively, follow the [guide to port your existing game to Rune](./how-it-works/existing-game.md) if you already have a game. 
+
+## Uploading & Playing In The App {#playing-the-game-in-app}
+
+Now that you have Tic Tac Toe running locally, it would be great to try playing it in the Rune app:
+
+```sh
+npm run upload
+```
+That's it. You'll now see your game inside Rune and can play it with your friends!
 
 ## Game Logic {#game-logic}
 
-Create a file named `logic.js` with a `setup` function that returns initial values for your `game` state that should be [synced across players](how-it-works/syncing-game-state.md). Add an action that modifies this `game` state and call `Rune.initLogic()` to initialize. For instance, to give all players a score and have an action that just increments the score:
+Rune games are split into two parts: logic & rendering. Let's take a look at the logic for the generated Tic Tac Toe game.
+
+You can find the game logic in the `logic.js` file. The `setup` function is responsible for creating an initial `game` state that's synced across players:
 
 ```js
-// logic.js
+function setup() {
+  const game = {
+    cells: new Array(9).fill(null), // 3x3 cell grid
+    lastMovePlayerId: null,
+    // ... rest of the game state
+  }
+  return game
```

Comment:
I know defining `game` is redundant, but IMO it makes it more clear that it's the same state used inside the actions etc.